src/server/helpers/html.js

```diff
@@ -6,20 +6,40 @@ import { Provider } from 'react-redux';
 import { conf } from '../helpers/config';
 
 const GAID = conf.get('GOOGLE_ANALYTICS_ID');
+const OPTID = conf.get('GOOGLE_OPTIMIZE_ID');
+const GTMID = conf.get('GOOGLE_TAG_MANAGER_ID');
 
-const googleTagManager = GAID ?
+const googleAnalytics = GAID && OPTID ?
+  <style>.async-hide { opacity: 0 !important} </style>
```

Comment:
This unfortunately doesn't work in react JSX. `{ opacity : 0 ... }` fails to parse as it's treated as JSX value rather then CSS string.
Also JSX element (`googleAnalytics` here) needs to have one single root node (here we have 3: style, script, script).

Also inline scripts won't work as JSX will try to parse them rather then render as strings.

The simplest way to do this would be something similar that is done to `googleTagManager` below (using `dangerouslySetInnerHTML`).

Reply:
debddd8